### ISSUE TITLE
Adwords api authentication with service account

### ISSUE THREAD

**Issue by vyurbonas** [googleads/google-ads-dotnet#192]
**Actions taken**
This is my App.config section :

```
<add key="OAuth2Mode" value="SERVICE_ACCOUNT" />
<add key="OAuth2PrnEmail" value="*****" />
<add key="OAuth2SecretsJsonPath" value="******.json" />
<add key="DeveloperToken" value="***********"/>
<add key = 'LoginCustomerId' value = '*********' />
```

**Expected result**
Authentication with service account without any interaction from the user

**Actual result**
Got error:"Missing required parameter: refresh_token", Uri:""
 
** Most likely cause of error (suggested by support via email) and workaround:
Looks like reading the OAuth2Mode from App.config in this method.
https://github.com/googleads/google-ads-dotnet/blob/master/src/Config/GoogleAdsConfig.cs#L487

As a workaround, if you set
```
var client = new GoogleAdsClient();
client.Config.OAuth2Mode = OAuth2Flow.SERVICE_ACCOUNT;

```
Should work.

**Comment by AnashOommen** [comment]
I thought this made it into v3.0.0, but a bad GitHub merge caused this fix to drop. I'll get this live for v3.1.0.

**Comment by AnashOommen** [comment]
Fixed in v4.0.0.
